webviz_subsurface/plugins/_simulation_time_series/types/derived_ensemble_vectors_accessor_impl.py

```diff
@@ -63,10 +67,17 @@ def __init__(
             if expressions is not None
             else []
         )
-        self._resampling_frequency = (
-            resampling_frequency if self._provider.supports_resampling() else None
-        )
+
         self._relative_date = relative_date
+        self._resampling_options = (
+            resampling_options if self._provider.supports_resampling() else None
+        )
+
+        # Make date span union if calculating data relative to date
```

Comment:
Should we expand on this comment? Not immediately clear to me why data relative to date → we should take date unione.